hera_mc/cm_utils.py

```diff
@@ -16,6 +16,17 @@
 PAST_DATE = '2000-01-01'
 
 
+class utility_method:
```

Comment:
Is there a risk or impact of the way I did it?  I like it because it is so simple and flexible.  You can add arguments so compactly etc.  With the namespace you add_argument for everything, you need the '--'(or not?), etc.  Unless there is a reason to, I'd like to leave it.  Thoughts?